system/lib/libc.symbols

```diff
@@ -1,310 +1,541 @@
--------- C h_errno
--------- C __hwcap
```

Comment:
why is this diff so large? I think the symbols should be sorted - maybe you didn't sort them before applying?

Reply:
This is an as-is result from calling llvm-nm on new generated libc.bc. Is there some special process that one must do instead of dumping them? Perhaps there should be an automatic script that updates the symbol files from already built cache?

Reply:
We should probably add a script, but basically just pipe it through ` | sort`

Reply:
Ah gotcha, updated the symbols files (using embuilder to build them, then `llvm-nm ~/.emscripten_cache/asmjs/libc.bc | sort > system/lib/libc.symbols`